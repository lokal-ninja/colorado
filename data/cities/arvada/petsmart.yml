id: 2202039725
type: node
latitude: 39.79405
longitude: -105.08298
postcode: "80002"
city: Arvada
street: Wadsworth Bypass
housenumber: "5285"
phone: +1 303-456-1114
opening_hours: Mo-Sa 09:00-21:00; Su 10:00-19:00
website: https://www.petsmart.com/stores/us/co/arvada-store0252.html
