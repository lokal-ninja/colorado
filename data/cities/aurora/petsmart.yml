id: 363661146
type: way
latitude: 39.58234
longitude: -104.72096
postcode: "80016"
city: Aurora
street: "South Gartrell Road"
housenumber: 7350
phone: "+1 303-400-4778"
opening_hours: "Mo-Sa 09:00-21:00; Su 10:00-19:00"
website: "https://www.petsmart.com/stores/us/co/aurora-store1421.html"