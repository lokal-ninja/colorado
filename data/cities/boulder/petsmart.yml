id: 3145956030
type: node
latitude: 40.0191
longitude: -105.25192
postcode: "80301"
city: Boulder
street: "30th Street"
housenumber: 1850
phone: "+1 303-449-0201"
opening_hours: "Mo-Sa 07:00-21:00; Su 08:00-19:00"
website: "https://www.petsmart.com/stores/us/co/boulder-store1915.html"