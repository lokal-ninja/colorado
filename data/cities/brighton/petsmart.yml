id: 220899452
type: node
latitude: 39.95182
longitude: -104.78575
postcode: "80601"
city: Brighton
street: Prairie Center Parkway
housenumber: "2435"
phone: +1 303-637-9232
opening_hours: Mo-Sa 09:00-21:00; Su 10:00-19:00
website: https://www.petsmart.com/stores/us/co/brighton-store1855.html
