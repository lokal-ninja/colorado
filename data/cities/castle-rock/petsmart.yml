id: 12774559197
type: node
latitude: 39.40812
longitude: -104.86145
postcode: "80104"
city: Castle Rock
street: Milestone Lane
housenumber: "4654"
phone: +1-720-733-6282
opening_hours: Mo-Sa 09:00-21:00; Su 10:00-19:00
website: https://www.petsmart.com/stores/us/co/castle-rock-store1183.html
