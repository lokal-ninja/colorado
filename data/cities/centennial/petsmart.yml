id: 10162532283
type: node
latitude: 39.56812
longitude: -104.959
postcode: "80122"
city: Centennial
street: "South University Boulevard"
housenumber: 8222
phone: "+1-303-220-0215"
opening_hours: "Su 10:00-19:00; Mo-Sa 09:00-21:00"
website: ""
