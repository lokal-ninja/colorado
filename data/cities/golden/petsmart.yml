id: 92735977
type: way
latitude: 39.7222133
longitude: -105.19376190000001
postcode: "80401"
city: Golden
street: ""
housenumber: 
phone: ""
opening_hours: ""
website: ""
